### DISCUSSION TITLE
GPTK Required? How to Select?

### DISCUSSION THREAD

**Discussion by GryphonCoding** [italomandara/CXPatcher#175] [Q&A] [answered]
Hello, everyone. I'm trying to use CXPatcher 4.10 on Crossover 23.7.1 to add Apple's GPTK to CrossOver, (with the ultimate aim of running The Talos Principle 2 on Steam and avoiding the "directx 12" barrier).

I'm following the guide on CodeWeaver's website, but when it comes to adding GPTK to CXPatcher, I'm not seeing the options they are talking about. Obviously, I can't ask CodeWeavers for support, so I'm asking here and hoping someone can help.

On the CodeWeaver's guide it says 

> enable both “Enable External Resources” and “Allow repatch / upgrade.” Click “Locate External Resources,” search for the Game Porting Toolkit under the “Locations” section in the left panel, select it, and click “Open.”

I don't see any of these options in CXPatcher 4.10. I've looked at "advanced options", I've installed GSStreamer, which didn't seem to add anything, I've got the Apple GPTK downloaded, and I don't see other options.

I did try just patching CrossOver as-is, and it said it was successfully patched, but I still got the "requires directx 12" message when I tried to run the game.

Sorry if this seems basic, but I'm quite lost.

**Comment by italomandara** [comment, marked as answer]
GPTK (D3DMetal) is already included in the latest Crossover

**Reply by GryphonCoding** [reply]
Thanks! Just so I'm clear: GPTK is included in the latest _Crossover_ or _CXPatcher_?

**Reply by italomandara** [reply]
crossover we will re-enable GPTK installation if there's a new release of D3DMetal that's newer than the one built-in in crossover.
